packages/framework/composer.json

```diff
@@ -45,47 +45,60 @@
+        "ext-zip": "*",
+        "arvenil/ninja-mutex": "^0.4.1",
+        "bmatzner/jquery-bundle": "^2.2.2",
+        "bmatzner/jquery-ui-bundle": "^1.10.3",
         "commerceguys/intl": "0.7.4",
-        "craue/formflow-bundle": "3.0.3",
-        "doctrine/common": "2.8.1",
-        "doctrine/doctrine-bundle": "1.8.1",
-        "doctrine/doctrine-fixtures-bundle": "3.0.2",
+        "craue/formflow-bundle": "^3.0.3",
+        "doctrine/common": "^2.8.1",
+        "doctrine/annotations": "^1.6",
+        "doctrine/cache": "^1.7",
+        "doctrine/collections": "^1.5",
+        "doctrine/data-fixtures": "^1.3",
+        "doctrine/dbal": "^2.7",
+        "doctrine/doctrine-bundle": "^1.8.1",
+        "doctrine/doctrine-fixtures-bundle": "^3.0.2",
         "doctrine/orm": "dev-doctrine-260-with-ddc1960-hotfix-and-ddc4005-hotfix as 2.6.0",
-        "egeloen/ckeditor-bundle": "4.0.6",
-        "fp/jsformvalidator-bundle": "1.5.1",
-        "fzaninotto/faker": "1.7.1",
-        "helios-ag/fm-elfinder-bundle": "6.2.1",
-        "heureka/overeno-zakazniky": "2.0.6",
+        "egeloen/ckeditor-bundle": "^4.0.6",
+        "fp/jsformvalidator-bundle": "^1.5.1",
+        "fzaninotto/faker": "^1.7.1",
+        "gedmo/doctrine-extensions": "^2.4",
+        "helios-ag/fm-elfinder-bundle": "^6.2.1",
+        "heureka/overeno-zakazniky": "^2.0.6",
         "intaro/postgres-search-bundle": "dev-master#06625bdc1d",
-        "intervention/image": "2.3.14",
-        "jms/translation-bundle": "1.4.1",
-        "joschi127/doctrine-entity-override-bundle": "0.5.0",
+        "intervention/image": "^2.3.14",
+        "jms/metadata": "^1.6",
+        "jms/translation-bundle": "^1.4.1",
+        "joschi127/doctrine-entity-override-bundle": "^0.5.0",
+        "nikic/php-parser": "^3.1",
         "presta/sitemap-bundle": "^1.5.3",
-        "prezent/doctrine-translatable-bundle": "1.0.3",
-        "sensio/distribution-bundle": "5.0.21",
-        "sensio/framework-extra-bundle": "3.0.29",
+        "prezent/doctrine-translatable": "^1.2",
+        "prezent/doctrine-translatable-bundle": "^1.0.3",
+        "psr/log": "^1.0",
+        "sensio/distribution-bundle": "^5.0.21",
+        "sensio/framework-extra-bundle": "^3.0.29",
         "shopsys/migrations": "dev-master",
         "shopsys/form-types-bundle": "dev-master",
         "shopsys/plugin-interface": "dev-master",
         "shopsys/product-feed-interface": "dev-master",
-        "stof/doctrine-extensions-bundle": "1.3.0",
-        "symfony/assetic-bundle": "2.8.2",
-        "symfony/monolog-bundle": "3.1.2",
+        "stof/doctrine-extensions-bundle": "^1.3.0",
+        "swiftmailer/swiftmailer": "^6.0",
+        "symfony/assetic-bundle": "^2.8.2",
+        "symfony/monolog-bundle": "^3.1.2",
         "symfony/swiftmailer-bundle": "^3.2.2",
-        "symfony/symfony": "^3.4.8",
-        "symfony-cmf/routing": "2.0.3",
-        "symfony-cmf/routing-bundle": "2.0.3",
+        "symfony/symfony": "^3.4.0",
```

Comment:
Why did you rewrite ^3.4.8 to ^3.4.0 when you are using version ^3.4.8 in other packages?

Reply:
a mistake. Fixed